pkg/client/container.go

```diff
@@ -77,9 +95,30 @@ func (c Client) DeleteContainer(ctx context.Context, id *container.ID, opts ...C
 }
 
 func (c Client) GetEACL(ctx context.Context, id *container.ID, opts ...CallOption) (*eacl.Table, error) {
+	v, err := c.getEACL(ctx, id, true, opts...)
```

Comment:
Isn't `true` should be optional?

Reply:
The logic of `GetEACL` remained untouched. I do not find a challenge with complete disregard of the signature meaningful. In any case, such a scenario can be reproduced through `GetEACLWithSignature` method.